autoscaler/controllers/gateway/hpa.go

```diff
@@ -22,11 +22,12 @@ import (
 
 const (
 	memoryLimitPercentageForHPA = 75
+	cpuLimitPercentageForHPA    = 75
 )
 
 var (
-	minReplicas                = intPtr(1)
-	maxReplicas                = int32(10)
+	defaltMinReplicas          = intPtr(1)
+	defaultMaxReplicas         = int32(10)
```

Comment:
these value describes the same thing (replicas) but one is `intPtr` and the other is `int32`. should they both be `int32`?